src/components/App/App.jsx

```diff
@@ -1,13 +1,35 @@
+import React from "react";
 import { Button } from "../../common";
+import ResumeCreation from "../Resume/ResumeCreation.jsx";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 import "./App.scss";
 
 function App() {
   return (
-    <div className="container-btn">
-      <Button text="Resume" />
-      <Button text="Project" />
-    </div>
+    <Router>
+      <div className="content">
+        <Switch>
+          <Route exact path="/">
+            <div className="container-btn">
+              <Button text="Resume" />
+              <Button text="Project" />
+            </div>
+          </Route>
+          <Route exact path="/projectlist"></Route>
+          <Route exact path="/projectcreation"></Route>
+          <Route exact path="/projectdetail/:id"></Route>
+          <Route exact path="/resumelist"></Route>
+          <Route
+            className="layout-resume-creation"
+            exact
+            path="/resumecreation"
+            component={ResumeCreation}
+          ></Route>
+          <Route exact path="/resumedetail/:id"></Route>
```

Comment:
we need to think more about the naming of our routes. can you add a TODO comment?
```
// TODO revisit route names
```